cmd2/argparse_custom.py

```diff
@@ -872,7 +872,7 @@ def __init__(
         usage: Optional[str] = None,
         description: Optional[str] = None,
         epilog: Optional[str] = None,
-        parents: Sequence[argparse.ArgumentParser] = [],
+        parents: Sequence[argparse.ArgumentParser] = (),
```

Comment:
Using a mutable object in an initializer is generally frowned upon because that instance is shared for every call to the function. The validators complain.